graphdatascience/tests/integration/conftest.py

```diff
@@ -123,7 +123,7 @@ def clean_up(gds: GraphDataScience) -> Generator[None, None, None]:
     for graph_name in res["graphName"]:
         gds.graph.get(graph_name).drop(failIfMissing=True)
 
-    res = gds.beta.pipeline.list()
+    res = gds.pipeline.list() if gds.server_version() >= ServerVersion(2, 5, 0) else gds.beta.pipeline.list()
```

Comment:
If I was successful with the backwards-compat I mention above, then this would not be necessary.